common/lib/xmodule/xmodule/course_module.py

```diff
@@ -940,6 +946,10 @@ def set_grading_policy(self, course_policy):
         self.raw_grader = grading_policy['GRADER']  # used for cms access
         self.grade_cutoffs = grading_policy['GRADE_CUTOFFS']
 
+    def set_default_certificate_available_date(self):
+        if (not self.certificate_available_date) and self.end:
+            self.certificate_available_date = self.end + timedelta(days=2)
```

Comment:
Nit: would it be worth the overhead of making this 2-day default delay a configurable setting?